GitCommands/Git/GitModule.cs

```diff
@@ -595,6 +596,30 @@ public void SaveBlobAs(string saveAs, string blob)
             stream.Write(blobData, 0, blobData.Length);
         }
 
+        public void SaveTreeAs(string saveAs, string hash)
+        {
+            string archivePath = Path.Join(saveAs.SubstringUntilLast('\\'), $"{hash}.zip");
+
+            if (!File.Exists(archivePath))
+            {
+                GitArgumentBuilder args = new("archive")
+                {
+                    $"--format zip",
+                    $"--output \"{archivePath.ToNativePath()}\"",
+                    $"{hash}",
+                    "-0"
+                };
+
+                using var process = _gitCommandRunner.RunDetached(args);
```

Comment:
I think nothing from `GitCommandRunner` is used:
```suggestion
                using IProcess process = _gitExecutable.Start(args);
```